packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -1190,7 +1190,7 @@ export class EditorStore implements CommandRegistrar {
           // by the `closeState()` call later
           return true;
         }
-        return !generatedChildrenElements.includes(elementState.element);
+        return !elementsToDelete.includes(elementState.element);
```

Comment:
shouldn't this be a new condition including both `generatedChildrenElements` and `elementsToDelete`?

Reply:
`elementsToDelete` includes `generatedChildrenElements`